humanguide/other-aspects/03-importing-data.rst

```diff
@@ -0,0 +1,10 @@
+Importing Data
+==============
+
+Due to the isolation between apps, when interchange of data between apps
+is needed, is done through the Content Hub.
```

Comment:
"Interchange of data between apps is done through Content Hub. This improves isolation between apps."

Maybe link to the content hub page too.